app/elm/Data/Maps.elm

```diff
@@ -0,0 +1,15 @@
+port module Data.Maps exposing (..)
+
+import Model exposing (..)
+
+
+port initMap : Marker -> Cmd msg
+
+
+port updateMarkers : List Marker -> Cmd msg
+
+
+eventMarkers : List Event -> List Marker
```

Comment:
This one is a bit tricky as to where it should live, I'd even be tempted to put it in the `Data.Events` module (even though it's related to the map)